backend/devices/bitbox/relay/channel.go

```diff
@@ -171,14 +171,8 @@ func (channel *Channel) waitForValue(duration time.Duration, name string) (strin
 
 // WaitForScanningSuccess waits for the given duration for the scanning success from the mobile.
 // Returns nil if the scanning success was retrieved from the relay server and an error otherwise.
-func (channel *Channel) WaitForScanningSuccess(duration time.Duration) error {
-	value, err := channel.waitForValue(duration, "id")
-	if err != nil {
-		return err
-	} else if value != "success" {
-		return errp.New("Scanning unsuccessful")
-	}
-	return nil
+func (channel *Channel) WaitForScanningSuccess(duration time.Duration) (string, error) {
```

Comment:
delete this function? it is too trivial of a wrapper now. The docstring is also outdated.

Reply:
Should I delete the other WaitFor* functions as well?

Reply:
Hmm good point. I would not, as they are nicely documented and seem to mirror the relay server api.

In that case, this one should also be kept then, but with updated documentation.

Or keep the function as it was, returning an error only, but have the caller just inspect the error to determine what action to take (ErrScanningFailed, ErrPullFailed, ...).

Reply:
I prefer just updating the documentation then, since the other WaitFor* functions use a similar pattern.